package.json

```diff
@@ -2,11 +2,13 @@
   "name": "docs",
   "version": "0.0.1",
   "scripts": {
-    "lint-markdown": "node ./scripts/markdown_seo_lint.js"
+    "lint-markdown": "node ./scripts/markdown_seo_lint.js",
+    "lint-links": "node ./scripts/link_linter.js"
   },
   "dependencies": {
     "broken-link-checker": "^0.7.8",
     "clipboard-polyfill": "^2.8.1",
+    "node-html-parser": "^1.1.16",
```

Comment:
Is this still needed? I don't see it being used anywhere.

Reply:
Removed